public/manifest.webextension-shim.json

```diff
@@ -1,5 +1,11 @@
 {
     "$schema": "https://dimensiondev.github.io/webextension-polyfill/schema/manifest.json",
-    "external-uri": ["^wc:", "^https?://arweave\\.net"],
+    "external-uri": ["^https?:\\/\\/arweave\\.net"],
+    "universal-link": [
+        "^wc:",
+        "^https?:\\/\\/.{6,26}\\/wc\\?uri=wc:",
+        "^.{3,18}:\\/\\/wc\\?uri=wc:",
```

Comment:
what is this?

Reply:
from https://docs.walletconnect.org/mobile-linking

see https://github.com/WalletConnect/walletconnect-monorepo/blob/next/packages/helpers/mobile-registry/registry.json